app/forms/registration_form.rb

```diff
@@ -33,55 +33,13 @@ def cocina_model
 
     structural = {}
     structural[:isMemberOf] = [params[:collection]] if params[:collection].present?
-    case content_type_tag
-    when 'Book (ltr)'
-      structural[:hasMemberOrders] = [{ viewingDirection: 'left-to-right' }]
-    when 'Book (rtl)'
-      structural[:hasMemberOrders] = [{ viewingDirection: 'right-to-left' }]
-    end
+    structural[:hasMemberOrders] = [{ viewingDirection: params[:viewing_direction] }] if params[:viewing_direction].present?
+
     model_params[:structural] = structural
     model_params[:administrative][:partOfProject] = params[:project] if params[:project].present?
 
     Cocina::Models::RequestDRO.new(model_params)
   end
 
-  # All the tags from the form except the project and content type, which are handled specially
-  def administrative_tags
-    params[:tag].filter { |t| !t.start_with?('Process : Content Type') }
-  end
-
```

Comment:
there is no longer a `Process : Content Type` tag coming from the reg form since we don't need it to compute the content type, so need to strip it anymore